etc/config/ispc.amazon.properties

```diff
@@ -47,11 +49,6 @@ compiler.ispc191.semver=1.9.1
 compiler.ispc-trunk.exe=/opt/compiler-explorer/ispc-trunk/bin/ispc
 compiler.ispc-trunk.semver=(trunk)
 compiler.ispc-trunk.isNightly=true
-compiler.ispc-templates_new-trunk.exe=/opt/compiler-explorer/ispc-templates_new-trunk/bin/ispc
```

Comment:
Be aware anyone who has saved a short link to this compiler will now have a broken link (it will likely redirect to the default compiler). If you want to steer to a specific compiler we can set up an alias for `ispc-template_new-trunk` to point to a specific ispc.